sync_engagement_db_to_coda.py

```diff
@@ -17,6 +17,8 @@
                         help="Logs the updates that would be made without updating anything.")
     parser.add_argument("--incremental-cache-path",
                         help="Path to a directory to use to cache results needed for incremental operation.")
+    parser.add_argument("-s", "--skip-updating-coda-users-and-code-schemes", action="store_true",
+                        help="Whether to skip updating coda users and code schemes")
```

Comment:
I take your point but that isn’t the way I see it; instead, I think we cannot corrupt the analysis files because we cannot generate analysis files without the code schemes and also the default behaviour is to update the datasets in both stages.

Reply:
I think this is ok. As Daniel says, it shouldn't affect analysis files, and because it's an opt-in optimisation it won't affect any individual stage run manually. It just removes some duplicate work when running the pipeline in full.